ledger/storage/leveldb/leveldb.go

```diff
@@ -96,9 +99,28 @@ func InitDB() (*LevelLedger, error) {
 		return nil, err
 	}
 
-	return &LevelLedger{
+	var zeroID record.ID
+	copy([]byte(zeroRecordBinary)[:record.IDSize], zeroID[:])
+	ledger := LevelLedger{
 		ldb: db,
-	}, nil
+		zeroRef: record.Reference{
+			Domain: record.ID{}, // TODO: fill domain
+			Record: zeroID,
+		},
+	}
+	_, err = db.Get([]byte(zeroRecordHash), nil)
+	switch err {
```

Comment:
@andreyromancev @ita-sammann Error checking in this way is not idiomatic go code. I would like to see this part rewritten

Reply:
It makes sense. Let's use if/else if/else in similar cases.